edb/schema/pointers.py

```diff
@@ -1191,12 +1191,37 @@ def canonicalize_attributes(
             self.set_attribute_value(
                 'is_derived', True
             )
+            if isinstance(self, inheriting.AlterInheritingObject):
+                self._propagate_is_derived(schema, context, True)
 
             if context.declarative:
                 self.set_attribute_value(
                     'declared_overloaded', True
                 )
 
+        # If we *were* an aliased computed but are changing to not be
+        # one, we need to update our base and is_derived to not be.
+        elif (
+            isinstance(self, inheriting.AlterInheritingObject)
+            and self.has_attribute_value('expr')
+            and (
+                (cur_base := self.scls.get_bases(schema).objects(schema)[0])
+                and (subj := cur_base.get_subject(schema))
+                and subj == self.scls.get_subject(schema)
```

Comment:
This checks if my subject is also the subject of my first base.

So it checks that I'm haven't been rebased to a different subject?

Why is this here?